src/lamp_py/performance_manager/alerts.py

```diff
@@ -331,12 +318,16 @@ def unix_to_est(unix_time: Optional[int]) -> Optional[datetime]:
     if unix_time is None or pandas.isna(unix_time):
         return None
 
-    # Create a timezone-aware datetime object in UTC
```

Comment:
update this function to return a datetime that is localized to boston but without the tz info.